### PR TITLE
[Merged by Bors] - Add bors.toml

### PR DESCRIPTION
**Pull request by AlanGriffiths**

### CODE REVIEW

**Review by Saviq — APPROVED** [review]
Right :)

bors merge

**Comment by Saviq** [comment]
Ah, we need something to gate on, first. Copy the snap workflow from somewhere else and add `status = [ "Snap" ]`?

**Comment by AlanGriffiths** [comment]
> Ah, we need something to gate on, first. Copy the snap workflow from somewhere else and add `status = [ "Snap" ]`?

@Saviq that needs a secret. Do you have the secret to adding them?

File: .github/workflows/snap.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      with:
+        snapcraft_token: ${{ secrets.SNAPCRAFT_TOKEN }}
+
+    - name: Check out code
+      uses: actions/checkout@v2
+
+    - name: Patch PPA and version
+      run: |
+        sed -i 's@mir-team/release@mir-team/dev@' snap/snapcraft.yaml
+        sed -i 's@rev-list --count@rev-parse --short@' snap/snapcraft.yaml
```
Comment (Saviq) [inline .github/workflows/snap.yml]: This isn't needed here.

**Review by Saviq — CHANGES_REQUESTED** [review]

**Review by Saviq — APPROVED** [review]
bors merge

**Pull request closed without merging**